clj-refactor.el

```diff
@@ -155,6 +155,14 @@ as can be."
   :group 'cljr
   :type 'boolean)
 
+(defcustom cljr-suppress-outside-project-warning nil
```

Comment:
I think `cljr-suppress-no-project-warning` might be a bit more self-explanatory.

Reply:
Good idea. Changed.